ibis/backends/bigquery/__init__.py

```diff
@@ -231,6 +231,45 @@ def project_id(self):
     def dataset_id(self):
         return self.dataset
 
+    def create_schema(
+        self,
+        name: str,
+        database: str | None = None,
+        force: bool = False,
+        collate: str | None = None,
+        **options: Any,
+    ) -> None:
+        code = "CREATE SCHEMA"
+        if force:
+            code += " IF NOT EXISTS"
```

Comment:
the wording is hard to parse for me, but https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#arguments says

```
IF NOT EXISTS: If any dataset exists with the same name, the CREATE statement has no effect.
```

Should we interpret `force` here to mean "don't error" or "overwrite" (or something else)?

Reply:
looking at `drop_schema` I think it's "don't error" -- and I'm having deja vu that we've discussed this at some point in the past.  

I think this is probably how `exists_ok` became a keyword argument in pathlib, but I don't know that we want to copy that.

Reply:
I'd rather keep it consistent with the current interfaces, which all use `force` even if `force` is suboptimal.